### PR TITLE
Upgrade crazy-max/ghaction-import-gpg GitHub Action to v3

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## WHAT
I've upgrade to `v3` with breaking changes.

## WHY
Needs to maintain this repo.

## Ref
* [Update crazy-max/ghaction-import-gpg action](https://github.com/hasuracommunity/terraform-provider-hasura/pull/12)
* [CHANGELOG, crazy-max/ghaction-import-gpg](https://github.com/crazy-max/ghaction-import-gpg/blob/master/CHANGELOG.md)

### CODE REVIEW

File: .github/workflows/release-provider.yml [hunk 1]
```diff
@@ -46,10 +46,10 @@ jobs:
             ${{ runner.os }}-go-
       - name: Import GPG key
         id: import_gpg
-        uses: crazy-max/ghaction-import-gpg@v2
-        env:
-          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-          PASSPHRASE: ${{ secrets.PASSPHRASE }}
+        uses: crazy-max/ghaction-import-gpg@v3
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
```
Comment (dragon3) [inline .github/workflows/release-provider.yml:51]: 😎 
```suggestion
          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
```

https://github.com/crazy-max/ghaction-import-gpg/blob/master/CHANGELOG.md#300-20200906

> - Move `GPG_PRIVATE_KEY` env var to `gpg-private-key` input
> - Move `PASSPHRASE` env var to `passphrase` input

**Review by dragon3 — CHANGES_REQUESTED** [review]

File: .github/workflows/release-provider.yml (hunk 1 above)
Comment (KeisukeYamashita) [reply .github/workflows/release-provider.yml:51]: Oh, nice catch, Thank you 🙇

**Review by dragon3 — APPROVED** [review]
LGTM 👍

**Comment by KeisukeYamashita** [comment]
Thank you!

**Pull request merged**